library/koryphe-library/src/main/java/uk/gov/gchq/palisade/policy/tuple/PalisadeTuple.java

```diff
@@ -38,8 +38,8 @@
 public class PalisadeTuple implements Tuple<String> {
     public static final String RECORD_NAMESPACE = "Record";
     public static final String NAMESPACE_MSG =
-            User.NAMESPACE + ", " + Context.NAMESPACE + " or " + RECORD_NAMESPACE + ". "
```

Comment:
Can we return these back to using the .NAMESPACE rather than hardcoding the values, given they are still in the korphye library User class